blocks/carousel/carousel.css

```diff
@@ -0,0 +1,518 @@
+  padding: 0 15px;
+  margin-bottom: 40px;
+  height: 395px;
+  display: flex;
+  max-width: 1440px;
+  width: 100%;
+  position: relative;
+}
+
+@media (max-width: 1024px) {
+  .landing-page main .carousel.block.wedges {
+    margin: 100px 0 40px;
+  }
+
+  .landing-page main .carousel.block .carousel-main-wrapper {
+    height: auto;
+  }
+}
+
+.landing-page main .carousel.block.wedges .carousel-main-wrapper {
+  max-width: 1630px;
+  padding: 0;
+  margin-bottom: 55px;
+  overflow: hidden;
+  height: auto;
+}
+
+.landing-page main .carousel.block .controls {
+  position: relative;
+  width: 100%;
+  max-width: 1440px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.landing-page main .carousel.block.wedges .controls {
+  max-width: 1630px;
+}
+
+@media (max-width: 779px) {
+  .landing-page main .carousel.block .controls {
+    display: none;
+  }
+}
+
+.landing-page main .carousel.block .controls button {
+  border: none;
+  background-color: #151517;
+  background-image: url("/icons/carousel-next-arrow.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 50%;
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  left: 0;
+  z-index: 1;
+  top: 96px;
+}
+
+.landing-page main .carousel.block.wedges .controls button {
+  top: 170px;
+  left: -16px;
+}
+
+.landing-page main .carousel.block .controls button.hidden {
+  visibility: hidden;
+}
+
+.landing-page main .carousel.block .controls button.right-button {
+  left: auto;
+  right: -12px;
+}
+
+.landing-page main .carousel.block .controls button.left-button {
+  transform: rotate(180deg);
+}
+
+.landing-page main .carousel.block .carousel-main-wrapper .carousel-frame {
+  display: flex;
+  gap: var(--carousel-card-gap);
+  height: 100%;
+  width: 100%;
+  transition: all 0.25s ease 0s;
+}
+
+.landing-page main .carousel.block .carousel-main-wrapper.grabbed .carousel-frame {
+  transition: none;
+}
+
+.landing-page main .carousel.block .carousel-main-wrapper .carousel-frame .carousel-item {
+  width: calc(25% - var(--carousel-card-gap) + 4px);
+  display: flex;
+  flex-direction: column;
+  border: 0.5px solid #e5e5e5;
+  background-color: #fff;
+  text-decoration: none;
+  flex-shrink: 0;
+}
+
+.landing-page main .carousel.block.wedges .carousel-main-wrapper .carousel-frame .carousel-item {
+  background: none;
+  border: none;
+  width: calc(16.66% - var(--carousel-card-gap) + 6px);
+  position: relative;
+  height: auto;
+}
+
+@media (max-width: 1280px) {
+  .landing-page main .carousel.block.wedges .carousel-main-wrapper .carousel-frame .carousel-item {
+    width: calc(33.33% - var(--carousel-card-gap) + 6px);
+  }
+}
+
+@media (max-width: 779px) {
+  .landing-page main .carousel.block.wedges .carousel-main-wrapper {
+    padding: 0;
+  }
+
+  .landing-page main .carousel.block.wedges .carousel-main-wrapper .carousel-frame .carousel-item {
+    width: 100%;
+  }
+}
+
+.landing-page
```

Comment:
Can you simplify the multiline selectors maybe ? IMO specifying main is not necessary as long as you have a specific top level selector like '.carousel'. 
Also I think carousels only appear on landing-pages so we probably don't need that selector neither ...

Reply:
I removed '.landing-page main' from all the CSS selectors